From f24c2c111984e918f3f142f05e7cd7d0bd609cbc Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Fri, 8 Feb 2013 14:41:25 -0600
Subject: [PATCH 417/486] ENGR00238059: iMX5x: Add power HAL to the list of
 build targets

Add imx5x power HAL to the list of build targets.

Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 8b98bd7..31f7ba2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,5 +1,5 @@
 common_imx_dirs := libsensors libgps lights wlan libbt-ath3k
-mx5x_dirs := $(common_imx_dirs) mx5x/audio mx5x/libcopybit mx5x/libgralloc  mx5x/hwcomposer mx5x/libcamera
+mx5x_dirs := $(common_imx_dirs) mx5x/audio mx5x/libcopybit mx5x/libgralloc  mx5x/hwcomposer mx5x/libcamera mx5x/power
 mx6_dirs := $(common_imx_dirs) alsa mx6/libgralloc_wrapper mx6/hwcomposer mx6/power
 
 ifeq ($(TARGET_BOARD_PLATFORM),imx6)
-- 
1.8.0

